File: docs/vsto/codesnippet/CSharp/Trin_VstcoreActionsPaneWordCS/ShowPropertiesControl.cs
using System;
using System.Windows.Forms;

namespace Trin_VstcoreActionsPaneWordCS
{
    partial class ShowPropertiesControl : UserControl
    {
        //---------------------------------------------------------------------
        //<Snippet28>
        public ShowPropertiesControl()
        {
            InitializeComponent();
            this.tableProperties.Click += new System.EventHandler(this.tableProperties_Click);
        }
        //</Snippet28>


        //---------------------------------------------------------------------
        //<Snippet27>
        private void tableProperties_Click(object sender, System.EventArgs e)
        {
            object timeout = 0;

            Globals.ThisDocument.Application.Dialogs[
                Microsoft.Office.Interop.Word.WdWordDialog.wdDialogTableProperties]
                    .Show(ref timeout);
        }
        //</Snippet27>
    }
}

File: docs/vsto/codesnippet/VisualBasic/Trin_VstcoreActionsPaneWordVB/ShowPropertiesControl.vb
Public Class ShowPropertiesControl

    '--------------------------------------------------------------------------
    '<Snippet27>
    Private Sub tableProperties_Click(ByVal sender As System.Object, ByVal e As System.EventArgs) _
        Handles tableProperties.Click

        Globals.ThisDocument.Application.Dialogs( _
            Word.WdWordDialog.wdDialogTableProperties).Show()
    End Sub
    '</Snippet27>
End Class
